### DISCUSSION TITLE
macOS PR contributors needed

### DISCUSSION THREAD

**Discussion by ckerr** [transmission/transmission#3225] [General]
This has been discussed in a few PRs, but I thought I'd make a new discussion thread for it (a) for visibility and (b) so that I'll have a URL to link to if the issue comes up again in future PRs.

## The Problem

macOS client PRs are going  unreviewed and/or stuck in limbo without ever getting an 'Approve' or 'Request changes'

## The Reason

There's not currently a primary person to approve / reject ideas for the macOS client. I'm the most active dev on Transmission, but I'm not a mac person and don't have a sense of idiomatic mac UI. @livings124 has done this in the past (and is still welcome!) but has very limited availability.

## The Solution

Happily, there are a lot of people who have volunteered macOS code recently: @Coeur, @DevilDimon, @fxcoudert, @Oleg-Chashko, @sweetppro. I've added you all to the collaborator list. Since there's currently no higher authority, you each have a decent amount of power to approve or request changes on macOS PRs.

So use this power. Review each other's PRs and request reviews for your own PRs. I'm looking for forward progress, for consensus via code reviews, and to keep mac PRs from languishing.

Don't get too crazy though. Even if you  find someone to approve it, I'm not merging your new Joker Color Theme PR. :green_heart: :purple_heart: 

Also, if you're not on the list above but want to contribute, you are welcome too!

**Comment by nevack** [comment]
@ckerr Can I join collaborator list? I have contributed to macOS client some time ago and also actively review all macOS-related changes. Thanks!
